scripts/west_commands/run_common.py

```diff
@@ -142,8 +142,10 @@ def add_parser_common(command, parser_adder=None, parser=None):
                        help=argparse.SUPPRESS)
     group.add_argument('-r', '--runner',
                        help='override default runner from --build-dir')
-    group.add_argument('--skip-rebuild', action='store_true',
+    group.add_argument('-s', '--skip-rebuild', "--no-rebuild", action='store_true',
                        help='do not refresh cmake dependencies first')
+    group.add_argument('--force-rebuild', '--rebuild', action='store_true',
```

Comment:
remove the `--force-rebuild`.

We should just have `--rebuild` / `--no-rebuild` as the primary flags. And then we're keeping `--skip-rebuild` for backwards compatibility reasons.
```suggestion
    group.add_argument('--rebuild', action='store_true',
```

Reply:
I think we need `default=None` here, so that we can determine later if the user has specified the argument (`True`/`False`) or if it was not specified (`None`).

Reply:
> I think we need default=None here,...

I think so too, otherwise how would we know when to consider/ignore the `west config`? See below.